### ISSUE TITLE
Missing package torch in online tutorials

### ISSUE THREAD

**Issue by nitzsche-fzi** [neuromorphs/tonic#254]
I tried to run the online tutorials on mybinder as linked in the readme (https://mybinder.org/v2/gh/neuromorphs/tonic/main?labpath=docs%2Ftutorials), but the environment seems to lack pytorch. Any import of torch fails:

![grafik](https://github.com/neuromorphs/tonic/assets/88715347/9e56abe9-1da0-4363-80b1-41636870dc5c)

**Comment by biphasic** [comment]
Hm Binder cannot build the image for me, that's weird. I tried some days ago and again today 

```
Found built image, launching...
Launching server...
Server requested
2023-06-14T14:35:19.489884Z [Normal] Successfully assigned ovh2/jupyter-neuromorphs-2dtonic-2dvqfchwu1 to user-202211a-node-dac61d
2023-06-14T14:35:20Z [Normal] Container image "jupyterhub/mybinder.org-tc-init:2020.12.4-0.dev.git.4289.h140cef52" already present on machine
2023-06-14T14:35:20Z [Normal] Created container tc-init
2023-06-14T14:35:20Z [Normal] Started container tc-init
2023-06-14T14:35:21Z [Normal] Pulling image "2lmrrh8f.gra7.container-registry.ovh.net/mybinder-builds/r2d-g5b5b759neuromorphs-2dtonic-e7577b:44e9aa6b5b4d114b990781ea40421cfb4409e6ef"
Launch attempt 1 failed, retrying...
```

it seems that you get past this point

**Comment by biphasic** [comment]
Okay on the third try, Binder managed to serve me with an environment. I replicated your problem. The issue is that currently all Tonic dependencies are defined in requirements.txt, but Binder uses that and only that to install the environment https://mybinder.readthedocs.io/en/latest/examples/sample_repos.html#notes
There might be a way of defining the Tonic requirements in setup.cfg, such that we can then include pytorch in the requirements.txt. I opened an issue here https://github.com/jupyterhub/mybinder.org-user-guide/issues/286
Until then, people will have to resort to using `!pip install torch torchvision` inside the notebooks
